src/main.rs

```diff
@@ -36,12 +36,13 @@ fn main() {
         Some(("test", sub_matches)) => cmd::test::execute(sub_matches),
         Some(("completions", sub_matches)) => (|| {
             let shell: Shell = sub_matches
-                .value_of("shell")
+                .get_one::<String>("shell")
+                .map(|s| s.as_str())
```

Comment:
This doesn't work and it currently panics. I recommend looking at the clap_complete documentation for how to set this up.